src/components/LinkManager/UserProfile/UserProfile.tsx

```diff
@@ -0,0 +1,26 @@
+import Image from "next/image";
+import styles from "./UserProfile.module.scss";
+import { SharedPageTypes } from "@/src/constants/types";
+
+interface UserProfile {
+  userProfile: SharedPageTypes;
+}
+
+export default function UserProfile({ userProfile }: UserProfile) {
+  const { name, owner } = userProfile;
+  return (
```

Comment:
return문 전의 개행은 특히 가독성에 많은 도움이 된다고 생각합니다~
```suggestion
  const { name, owner } = userProfile;
  
  return (
```